### ISSUE TITLE
When you edit a content of tab block, it opens the last tab of that block

### ISSUE THREAD

**Issue by Bojan-Svirkov** [cortezaproject/corteza#981]
## Description of the bug

_When I edit the content of a page block within a tab, it opens the last tab of that block configuration._

## Reproduction steps

1.	Create or edit a page
2.	Add a record list
3.	Save the record list
4.	Add a new tabs block
5.	Add the record list created above to the tabs block
6.	Save and exit
7.	Open the tabs block again in editing mode
8.	Click on the tab “Tabs” (the second tab) to see the list of configured tabs
9.	Click on the “edit” button of the only tab that’s in there (the record list)
10.	 Now the block configuration on the record list opens, but not in the “General” tab, but in the “Automation” tab.

## Expected behavior 

_When the record block configuration opens I expect it to always show the “General” tab (so, not the last tab)_

## Screenshot/Video

![Image](https://user-images.githubusercontent.com/53232926/227509867-00ef192a-2883-4fd4-af9b-b2b457b9f0da.png)

![Image](https://user-images.githubusercontent.com/53232926/227509898-596e3505-1067-4edf-8669-31006430a172.png)

## Version

Branch: 2023.3.0-dev.5.1 
Reporter: Lenny (Test fest)
Browser: Firefox

**Comment by katrinDY** [comment]
Investigate task and assess the feasibility. Timebox to 2hrs

**Comment by EmmyMay** [comment]
From investigating, I do not believe this is a problem coming from us but from the bootstrap-vue's library. I have created a sort of proof of concept that shows this is only reproducible if tabs are used in modals in a certain way. The best we could do right now if I am not wrong is find a way to recreate this issue and open it on bootstrap-vue's repo.

There is more information on my findings in the PR I opened for this. I will move this to review so that @Fajfa can take a look.
